source/agora/common/Amount.d

```diff
@@ -39,7 +39,7 @@ public struct Amount
     /// Number of units ('cents') per coins
     public static immutable Amount UnitPerCoin   = Amount(10_000_000, true);
     /// Maximum number of BOA coins that can ever be in circulation
-    public static immutable ulong  MaxCoinSupply = 500_000_000;
+    public static immutable ulong  MaxCoinSupply = 5_000_000_000;
```

Comment:
Wouldn't it be 4_950_000_000?

Reply:
Yes you are right. Updated.

Reply:
In fact this is from the other PR which I rebased this one on. Will update https://github.com/bosagora/agora/pull/2415

Reply:
https://github.com/bosagora/agora/pull/2415/commits/8851558aea35bb92168f88d03b6b33dfea1bd85c